gobblin-core/src/main/java/gobblin/writer/AvroHdfsTimePartitionedWriter.java

```diff
@@ -172,9 +170,11 @@ public void write(GenericRecord record) throws IOException {
     // If the path is in pathToWriterMap simply retrieve the writer, and write the record
     if (!this.pathToWriterMap.containsKey(writerOutputPath)) {
 
-      LOG.info("Creating a new DataWriter for path: " + new Path(this.baseFilePath, writerOutputPath));
+      Path writerOutputPathWithBase = new Path(this.baseFilePath, writerOutputPath);
+
+      LOG.info("Creating a new DataWriter for path: " + writerOutputPathWithBase);
```

Comment:
Yes. We probably should have had a better design of how different parts of the file path are stored in different places (in particular, redesign `WriterUtils.getWhateverPathOrNameOrDir` to make them more intuitive). But when looking at the name `baseFilePath` I have absolutely no idea what it refers to. So for now it's better to give it a better name.